gapic/src/main/com/google/gapid/models/Capture.java

```diff
@@ -80,9 +84,13 @@ public boolean isPerfetto() {
   protected ListenableFuture<Data> doLoad(File file) {
     if (!file.exists() || !file.canRead()) {
       return Futures.immediateFailedFuture(
-          new Exception("Trace file does not exist or is not accessible"));
+          new BadCaptureException("Trace file does not exist or is not accessible!"));
     } else if (file.length() == 0) {
-      return Futures.immediateFailedFuture(new Exception("Trace file is empty!"));
+      return Futures.immediateFailedFuture(
+          new BadCaptureException("Trace file is empty!"));
+    } else if (file.length() <= MIN_FILE_SIZE) {
+      return Futures.immediateFailedFuture(new BadCaptureException(
+          "Trace file is empty! Try capturing with buffering disabled."));
```

Comment:
The trace file is /technically/ not empty, maybe replace the error message by:
`"Trace file is too small to be valid! Try capturing with buffering disabled."`